cachito/web/api_v1.py

```diff
@@ -445,6 +456,7 @@ def patch_request(request_id):
         )
         try:
             bundle_dir.bundle_archive_file.unlink()
```

Comment:
Should be this block more error resilient and try to remove everything instead and reporting summary error instead of exception from first failure and skipping the rest of cleanup?

Reply:
What would be the benefit of the behavior "try to remove everything and report an error summary"?

Reply:
if my understanding is correct, this leaves some garbage behind in case unlink of bundle_archive_file failed

Reply:
Make sense. I'll implement this in a separate patch.